Project.toml

```
@@ -25,7 +25,7 @@ TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 CEnum = "0.2, 0.3, 0.4"
 DataFrames = "0.20, 0.21"
 Decimals = "0.4.1"
-DocStringExtensions = "0.8.0"
+DocStringExtensions = "0.8.0,0.9.1"
```

```suggestion
DocStringExtensions = "0.8.0, 0.9.1"
```
Standard form has spaces